.bazelrc

```
@@ -183,6 +184,8 @@ build:remote --spawn_strategy=remote,sandboxed,local
 build:remote --strategy=Javac=remote,sandboxed,local
 build:remote --strategy=Closure=remote,sandboxed,local
 build:remote --strategy=Genrule=remote,sandboxed,local
+# rules_rust is not remote runnable (yet)
+build:remote --strategy_regexp=_wasm_.*_rust=sandboxed,local
```

```suggestion
build:remote --strategy=Rustc=sandboxed,local
```

This was my oversight in envoy-wasm PR to actually enable Rust works in envoy CI.